common/src/keepassPlugin/databaseInterface/private/Keepass2DatabaseInterface.cpp

```diff
@@ -54,7 +55,8 @@ Keepass2DatabaseInterface::Keepass2DatabaseInterface(QObject *parent)
 Keepass2DatabaseInterface::~Keepass2DatabaseInterface()
 {
     qDebug("Destructor Keepass2DatabaseInterface");
-    delete m_Database;
+    m_Database.clear();
+    //delete m_Database;
```

Comment:
Line can be deleted. Same applies to below commented code.

Reply:
absolutely agree